Lib/test/test_inspect/test_inspect.py

```diff
@@ -3309,6 +3309,19 @@ def test(a, b, *, c, d):
                            ('d', ..., ..., "keyword_only")),
                           ...))
 
+        # With Placeholder
+        self.assertEqual(self.signature(partial(test, Placeholder, 1)),
+                         ((('a', ..., ..., "positional_or_keyword"),
+                           ('c', ..., ..., "keyword_only"),
+                           ('d', ..., ..., "keyword_only")),
+                          ...))
```

Comment:
If `a` is `positional_or_keyword`, it could be passed by keyword: `partial(test, Placeholder, 1)(a=1, c=2, d=3)`. But `Placeholder` only takes place for positional-only parameters. `a` should change its kind to `positional_only`.

Add also a test when `Placeholder` take place of var-positional parameter.

For example:
```
def test(a, *args, b): ...
partial(test, 1, 2, Placeholder, 3)
```